app/server.js

```diff
@@ -12,7 +12,7 @@ var PORT = 8090;
 var getToken = function(token, originalRes){
   var options = {
     host: 'api.onfido.com',
-    path: '/v2/jwt',
+    path: '/v2/jwt?referrer=*://*/*',
```

Comment:
We updated the server so you have to specify the referrer.

I could make this localhost I suppose - I will investigate.